devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy

```diff
@@ -86,6 +86,7 @@ def configure() {
 	}
 	commands << command(zwave.basicV1.basicGet())
 	response(commands)
+	refresh()
```

Comment:
you'd want a `response` around this too.

or `response(commands + refresh())` I think would work